download-arm64-libs.py

```diff
@@ -0,0 +1,50 @@
+"""Downloads the matching ARM64 Python libs for the running version of Python.
```

Comment:
This is only used by CI, right? If so, what do you think about putting it in .github and also updating the comment to indicate that's how it's used.

(Even if moving it there isn't a great idea, I think the comment makes sense, because the casual reader isn't going to understand whether they need to run it for a local build or not)

Reply:
Probably it'll only be used by CI, though I have referenced it from the build-env docs for doing an ARM64 targeted build. But since it's primarily for CI, I can move it and update the docs reference.